src/builders.ts

```diff
@@ -17,12 +17,14 @@ SPDX-License-Identifier: Apache-2.0
 Copyright (c) OWASP Foundation. All Rights Reserved.
 */
 
+import { existsSync } from 'node:fs'
+import * as path from 'node:path'
```

Comment:
the `node:` notation, this might not work on node14.0 -- which is a supported branch.
will add additional CI tests and have it tested

Reply:
test run including node 14.0.0

https://github.com/CycloneDX/cyclonedx-node-npm/actions/runs/7114640707/job/19369091011?pr=1129